cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2063,4 +2063,11 @@ public static final class MetadataConsumer {
     public static final String METADATA_CONSUMER_EXTENSIONS_ENABLED_LIST = "metadata.consumer.extensions.enabled.list";
     public static final String METADATA_CONSUMER_EXTENSIONS_DIR = "metadata.consumer.extensions.dir";
   }
+
+  /**
+   * Configuration for source control management
+   */
+  public static final class SourceControl {
+    public static final String GIT_INTEGRATION_ENABLED = "scm.git.enable";
```

Comment:
My suggestion is 'feature.source.control.management.enabled'

Reply:
I also see we use Feature enum for adding feature flags. What is the recommendation here vis-a-vis config vs feature flag ?
cc: @arjan-bal @de-lan